lua/ulx/modules/sh/cfc_refund.lua

```diff
@@ -0,0 +1,37 @@
+local CATEGORY_NAME = "Refund"
+
+local playerKills = {}
+
+hook.Add( "PlayerInitialSpawn", "OnPlayerJoin", function(ply)
+	playerKills[ply] = {}
+end)
+
+hook.Add( "PlayerDisconnected", "OnPlayerDisconnect", function(ply)
+	playerKills[ply] = nil
+
+	for k, killData in pairs( playerKills )
+		killData[ply] = nil
+	end
+end)
+
+hook.Add( "PlayerDeath", "OnPlayerKill", function( ply, inflictor, attacker )
+	local playerKillsOnVictim = playerKills[ply][attacker]
+
+	playerKillsOnVictim = (playerKillsOnVictim or 0) + 1
+end)
+
+local function playerRefunds( callingPlayer, targetPlayers )
+
+	for victim, killsFromPly in pairs( playerKills[targetPlayers] ) do
```

Comment:
if  you changed the argument type to be a single player this variable name should be changed to reflect that
so instead of `targetPlayers` it should be `targetPlayer` because its not a single player